packages/web-client/app/components/workflow-thread/index.css

```diff
@@ -1,11 +1,29 @@
+.workflow-thread {
+  --inner-max-width: 36.6rem; /* ~585px */
+  --outer-max-width: 43.75rem; /* 700px; */
+}
+
 .workflow-thread__status {
   color: var(--boxel-dark);
   font: 600 var(--boxel-font-sm);
   letter-spacing: var(--boxel-lsp);
   text-align: center;
 }
 
+.workflow-thread__card > .action-card-container {
+  max-width: var(--outer-max-width);
+  box-shadow: 0 15px 30px rgba(0, 0, 0, 0.15);
+  transition: all var(--boxel-transition);
+}
+
+.workflow-thread__card--view-only > .action-card-container {
```

Comment:
`view-only` isn't accurate here. You can often go back and edit things in a workflow. How about `is-complete`, same as the argument that controls whether this class is applied?

Reply:
Would also replace `.workflow-thread__card > .action-card-container` and `.workflow-thread__card--view-only > .action-card-container` with `workflow-thread__card` and `workflow-thread__card--view-only`, and remove `box-shadow`. Box shadow styling is repeated here and in action container, and we want the margins to apply to the containing element to prevent the card exceeding the bounds of the thread:

![image](https://user-images.githubusercontent.com/39579264/117140762-56794600-ade0-11eb-8913-0b74bb167bb1.png)

Reply:
There's also an issue with the margins that I think we can iron out later - the right margin is not really present once the viewport shrinks a bit.

![image](https://user-images.githubusercontent.com/39579264/117140930-89bbd500-ade0-11eb-9e51-a1e777a34a91.png)

Reply:
I couldn't replicate this, but I think this is actually closer to how it's supposed to look in smaller viewports. We can check while doing the mobile version

Reply:
> There's also an issue with the margins that I think we can iron out later - the right margin is not really present once the viewport shrinks a bit.
> 
> ![image](https://user-images.githubusercontent.com/39579264/117140930-89bbd500-ade0-11eb-9e51-a1e777a34a91.png)

I couldn't replicate this, but I think this is actually closer to how it's supposed to look in smaller viewports. We can check while doing the mobile version

Reply:
> Would also replace `.workflow-thread__card > .action-card-container` and `.workflow-thread__card--view-only > .action-card-container` with `workflow-thread__card` and `workflow-thread__card--view-only`, and remove `box-shadow`. Box shadow styling is repeated here and in `ActionCardContainer`, and we want the margins to apply to the containing element to prevent the card exceeding the bounds of the thread:
> 
> ![image](https://user-images.githubusercontent.com/39579264/117140762-56794600-ade0-11eb-8913-0b74bb167bb1.png)

The problem is, when the margin is only on the containing element, the latency causes the switch from one mode to the next to be janky. So I can add a note here to find a better way to implement the transition. Ember-animated would probably resolve this issue, but we haven't added it in yet.

Reply:
I'll push a different solution to this ^

Reply:
> Would also replace `.workflow-thread__card > .action-card-container` and `.workflow-thread__card--view-only > .action-card-container` with `workflow-thread__card` and `workflow-thread__card--view-only`, and remove `box-shadow`. Box shadow styling is repeated here and in `ActionCardContainer`, and we want the margins to apply to the containing element to prevent the card exceeding the bounds of the thread:
> 
> ![image](https://user-images.githubusercontent.com/39579264/117140762-56794600-ade0-11eb-8913-0b74bb167bb1.png)

This crop issue should be resolved now. Let me know if it's not.